docs/docs/detectors/6-unprotected-update-current-contract-wasm.md

```diff
@@ -1,17 +1,26 @@
-# Unprotected update of current contract wasm
+# Unprotected update current contract wasm
 
-### What it does
+## Description 
 
-It warns you if `update_current_contract_wasm()` function is called without a previous check of the address of the caller.
+- Category: `Authorization`
+- Severity: `Critical`
+- Detector: [`unprotected-update-current-contract-wasm`](https://github.com/CoinFabrik/scout-soroban/tree/main/detectors/unprotected-update-current-contract-wasm)
+- Test Cases: [`unprotected-update-current-contract-wasm-1`](https://github.com/CoinFabrik/scout-soroban/tree/main/test-cases/unprotected-update-current-contract-wasm/unprotected-update-current-contract-wasm-1) [`unprotected-update-current-contract-wasm-2`](https://github.com/CoinFabrik/scout-soroban/tree/main/test-cases/unprotected-update-current-contract-wasm/unprotected-update-current-contract-wasm-2) 
+
+It warns you if `update_current_contract_wasm()` function is called without a previous check of the address of the caller. If users are allowed to call `update_current_contract_wasm()`, they can intentionally modify the contract behaviour, leading to the loss of all associated data/tokens and functionalities given by this contract or by others that depend on it.
```

Comment:
Sacar la oración "If users are allowed..." dado que es la misma que está en la sección "Why is this bad?"